erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -1071,7 +1071,7 @@ var get_healthcare_services_to_invoice = function(frm) {
 		var patient = dialog.fields_dict.patient.input.value;
 		if(patient && patient!=selected_patient){
 			selected_patient = patient;
-			var method = "erpnext.healthcare.utils.get_healthcare_services_to_invoice";
+			var method = "healthcare.healthcare.utils.get_healthcare_services_to_invoice";
```

Comment:
We should be removing all healthcare-related code from Sales Invoices. This can be "plugged in" from the healthcare app hooks.

Reply:
Moved healthcare related custom js to https://github.com/frappe/healthcare/pull/6